src/main/java/com/icthh/xm/ms/dashboard/domain/UiData.java

```diff
@@ -0,0 +1,56 @@
+package com.icthh.xm.ms.dashboard.domain;
+
+import com.icthh.xm.ms.dashboard.domain.converter.MapToStringConverter;
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.ToString;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@Entity
+@Table(name = "ui_data")
+@ToString
+public class UiData implements Serializable {
```

Comment:
Should we also introduce 
**- widgetId
- widgetInstaneId** 
fields, to store configuration for following situations

```
userKey | widgetId | widgetInstanceId | description
null,         VALUE,      null                         - general widget configuration
null,         VALUE,      VALUE                   -  widget configuration for widget in specific screen 
(defualts for filter)
VALUE,         VALUE,      VALUE                   -  widget configuration for widget in specific screen for specific user
```